tests/test_other.py

```diff
@@ -1407,12 +1412,10 @@ def test_embed_file(self):
       }
     ''')
 
-    self.run_process([EMXX, 'main.cpp', '--embed-file', 'somefile.txt'])
-    self.assertContained('|hello from a file wi|', self.run_js('a.out.js'))
-
-    # preload twice, should not err
-    self.run_process([EMXX, 'main.cpp', '--embed-file', 'somefile.txt', '--embed-file', 'somefile.txt'])
-    self.assertContained('|hello from a file wi|', self.run_js('a.out.js'))
+    self.run_process([EMXX, 'main.cpp'] + args)
+    # run in node.js to ensure we verify that file preloading works there
+    result = self.run_js('a.out.js', engine=config.NODE_JS)
```

Comment:
I don't think you need `engine=config.NODE_JS` here.. its the default for all tests in this file.

Reply:
This is defending against the default possibly changing in the future. Seems safer to make it explicit in places where it is specifically needed?

Reply:
OK, I think we should come up with a better way to specify node-only tests in the future.   Maybe `@requires_node` or something like that.